File: Ch1-Fundamentals/0017-Print-and-Return/main.py
def print_first_return_second( arr ):
    if (len(arr) < 2):
        print("\n!!! ERROR: Array must have at least 2 values.")
        return None
    print("First Value:", arr[0])
    return arr[1]

arr = [4, 8, 12]
second_val = print_first_return_second(arr)
print("Second Value:", second_val)

arr2 = [2]
none_test = print_first_return_second(arr2)

File: Ch1-Fundamentals/0017-Print-and-Return/main.js
function printFirstReturnSecond( arr ) {
    if (arr.length < 2) {
        console.log("\n!!! ERROR: Array must have at least 2 values.")
        return undefined
    }
    
    console.log("First Value:", arr[0]);
    return arr[1];
}

let arr = [ 5, 10, 15];
let secondValue = printFirstReturnSecond(arr);
console.log("Second Value:", secondValue)

let arr2 = [ 5 ];
let undefinedTest = printFirstReturnSecond(arr2);